pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationService.java

```diff
@@ -137,6 +143,10 @@ public String authenticateHttpRequest(HttpServletRequest request) throws Authent
         }
     }
 
+    public String authenticateHttpRequest(HttpServletRequest request) throws AuthenticationException {
```

Comment:
Is this still used? It looks like it is used in `AbstractWebSocketHandler`. Was it intentional to skip updating that reference? Should we annotate this with deprecated?